packages/create-astro/src/actions/dependencies.ts

```diff
@@ -39,6 +44,45 @@ export async function dependencies(
 			},
 			while: () => install({ packageManager: ctx.packageManager, cwd: ctx.cwd }),
 		});
+
+		let add = ctx.add;
+		if (!add && !ctx.yes) {
+			let { adapter } = await ctx.prompt({
+				name: 'adapter',
+				type: 'select',
+				label: title('adapter'),
+				message: 'Add an adapter',
+				choices: [
+					{ value: undefined, label: 'None' },
```

Comment:
```suggestion
					{ value: undefined, label: 'none' },
```
*Maybe* lower-case "none" would be easier to read and distinguish from "Node", which look quite close. 😅  Also, "none" is not a proper name, so no need to capitalize here, and I think that's a good thing!

Reply:
Hmm, I admit that it looks weird to have None and Node next to each others, but it also looks weird in lower case, since every other options (both in this prompt and other) always start with an uppercase 🤔

Reply:
Perhaps we can rename `Node` to `Node.js`?